TramsDataApi/Factories/AcademyConversionProjectResponseFactory.cs

```diff
@@ -51,6 +51,13 @@ public class AcademyConversionProjectResponseFactory
 				MemberOfParliamentParty = academyConversionProject.MemberOfParliamentParty,
 				GeneralInformationSectionComplete = academyConversionProject.GeneralInformationSectionComplete,
 				RisksAndIssuesSectionComplete = academyConversionProject.RisksAndIssuesSectionComplete,
+				// Legal Requirements
+				LegalRequirementsSectionComplete = academyConversionProject.LegalRequirementsSectionComplete,
```

Comment:
Are there any unit tests to update?

Reply:
Yeah, just updated the 'expected' builder to now map legal req's.